torch/_meta_registrations.py

```diff
@@ -543,6 +543,30 @@ def linalg_inv_ex_meta(A: Tensor, check_errors: bool = False):
     return L, infos
 
 
+@register_meta([aten.linalg_ldl_factor_ex.default, aten.linalg_ldl_factor_ex.out])
+@out_wrapper("LD", "pivots", "info")
+def linalg_ldl_factor_ex_meta(
+    self: Tensor,
+    *,
+    hermitian: bool = False,
+    check_errors: bool = False,
+) -> Tuple[Tensor, Tensor, Tensor]:
+    squareCheckInputs(self, "torch.linalg.ldl_factor_ex")
+    checkFloatingOrComplex(self, "torch.linalg.ldl_factor_ex")
+
+    shape = self.shape
+
+    # prefer column major strides
+    LD = self.new_empty(shape)
+    LD.as_strided_(shape, make_contiguous_strides_for(shape, row_major=False))
```

Comment:
It's not that we prefer them, but LAPACK and cusolver ask for the inputs to be column_major. Also, same as in the other PR, why not `empty_strided`?